src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -0,0 +1,244 @@
+    .shape({
+      name: Yup.string().required("Volume name is required"),
+      project: Yup.string().required(),
+      pool: Yup.string().required(),
+    })
+    .test("deduplicate", "", async function (values) {
+      const { name, project, pool } = values;
+      const notFound = await checkDuplicateName(
+        name,
+        project || "",
+        controllerState,
+        `storage-pools/${pool}/volumes/custom`,
+      );
+
+      if (notFound) {
+        return true;
+      }
+
+      return this.createError({
+        path: "name",
+        message:
+          "A volume with this name already exist in the target project and storage pool",
+      });
+    });
+
+  const formik = useFormik<StorageVolumeDuplicate>({
+    initialValues: {
+      name: getDuplicatedVolumeName(volume),
+      project: volume.project,
+      copySnapshots: true,
+      pool: volume.pool,
+    },
+    enableReinitialize: true,
+    validationSchema,
+    onSubmit: (values) => {
+      const payload: Partial<LxdStorageVolume> = {
+        name: values.name,
+        type: "custom",
+        config: volume.config,
+        description: volume.description,
+        content_type: volume.content_type,
+        source: {
+          name: volume.name,
+          type: "copy",
+          pool: volume.pool,
+          volume_only: !values.copySnapshots,
+          // logic from the lxc source code.
+          // We should not set source.project if target project is the same as the source project
+          project:
+            values.project !== volume.project ? volume.project : undefined,
+          location: volume.location,
+        },
+      };
+
+      const existingVolumeLink = (
+        <Link
+          to={`/ui/project/${volume.project}/storage/pool/${volume.pool}/volumes/custom/${volume.name}`}
+          onClick={(e) => e.stopPropagation()}
+        >
+          <ItemName item={volume} />
+        </Link>
+      );
+
+      const targetProject =
+        values.project === formik.initialValues.project ? "" : values.project;
+
+      duplicateStorageVolume(payload, values.pool, targetProject)
+        .then((operation) => {
+          toastNotify.info(`Duplication of volume ${volume.name} started.`);
+          eventQueue.set(
+            operation.metadata.id,
+            () => notifySuccess(values.name, values.project, values.pool),
+            (msg) =>
+              toastNotify.failure(
+                "Volume duplication failed.",
+                new Error(msg),
+                existingVolumeLink,
+              ),
+          );
+          close();
+        })
+        .catch((e) => {
+          toastNotify.failure(
+            "Volume duplication failed.",
+            e,
+            existingVolumeLink,
+          );
+        });
+    },
+  });
+
+  // TODO: currently we don't support target cluster member selection due to a bug in lxd, we should enable it once the bug is fixed
+  // see issue:https://github.com/canonical/lxd/issues/14155
```

Comment:
```suggestion
  // TODO: currently we don't support target cluster member selection due to a bug in lxd,
  // we should enable it once the bug is fixed. Note, that cluster member selection is not relevant
  // for remote storage pools, as they are not tied to a lxd cluster member.
  // see issue: https://github.com/canonical/lxd/issues/14155
```
Adding a bit more detail

Reply:
I removed the comment and created an [issue](https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/930?assignee=712020%3Accb7b61f-3f6a-41ec-9935-a79519dada4e&selectedIssue=WD-15524) for it.